frontend/src/lib/components/surveys-page/SurveysTable.svelte

```diff
@@ -66,25 +63,25 @@
 				<td class="info-entry tooltip">
 					{#if entry.uses_cryptographic_module}
 						<i class="material-symbols-rounded">encrypted</i>
-						<span class="tooltip-text {innerWidth <= 1272 ? 'right' : 'left'}"
+						<span class="tooltip-text {innerWidth <= 1239 ? 'right' : 'left'}"
```

Comment:
we should avoid adding magic numbers, add consts or try to make it width ratio dependant

Reply:
true, named constants are always better